src/components/flyout/deviceDeletionFlyout.js

```diff
@@ -61,7 +61,7 @@ class DeviceDeletionFlyout extends React.Component {
             {lang.DELETE_DEVICE}
           </div>
           <div className="content-description">
-            {IsSimulated ? lang.NOT_ABLE_TO_DELETE : lang.DELETE_DEVICE_DESCRIPTION}
+            {lang.DELETE_DEVICE_DESCRIPTION}
```

Comment:
There should always be text on the delete page. The notification box should only appear when simulated devices are being ignored. See the design from https://github.com/Azure/pcs-remote-monitoring-webui/issues/584